workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -683,6 +699,7 @@ public struct Request
 
             public static Request CreateRequest(Improbable.Worker.Query.EntityQuery entityQuery, uint? timeoutMillis = null, Object context = null)
             {
+                Debug.LogWarning("Cannot create entity queries returning component data - dropping query.");
```

Comment:
This needs to be conditional on the type of the query - might need to do some type checking on the `resultType` inside the `EntityQuery`

Reply:
ah yes - check is now included